experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_k8s_job.yml

```diff
@@ -0,0 +1,106 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: disk-loss-
+  namespace: default
```

Comment:
I think we are not using the namespace field in other experiments. either remove this here or add the same in other experiments.

Reply:
Done